### PR TITLE
Nerfs carbonhell's cheap cuisine

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
tweak: Carbonhell cheap cuisine vendor is now available only as contraband in vending machines
tweak: The amount of cheap cusines per vendor has been reduced from 6 to 4
balance: Reduced carbon's butterdog slip time to 4 from 8 
/:cl:

[why]: op Roundstart 8 slips that were added in as a "meme" (powergame code that somehow got post headmin approval) are op, nuff said

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Jesus I didn't realise it was 8 seconds, that's mad

File: hippiestation/code/game/objects/items/cheapcuisine.dm [hunk 1]
```diff
@@ -95,6 +95,10 @@
 	bitesize = 2
 	foodtype = GROSS | TOXIC
 
+/obj/item/reagent_containers/food/snacks/butterdog/carbon/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/slippery, 40)
```
Comment (JohnGinnane) [inline hippiestation/code/game/objects/items/cheapcuisine.dm]: Won't this just add another slippery component?

Comment (ghost) [reply hippiestation/code/game/objects/items/cheapcuisine.dm]: no actually
components actually have the flag `COMPONENT_DUPE_HIGHLANDER` by default
which means it automatically deletes the old component and applies the new version

**Review by ghost — DISMISSED** [review]

**Comment by ghost** [comment]
cuhrazee
+1

**Comment by JohnGinnane** [comment]
@Carbonhell vote pls

**Comment by Manly-Dwarf** [comment]
+1

**Comment by MiniMeatwad** [comment]
+1

**Comment by Carbonhell** [comment]
Honestly remove the slipping component altogether. There are way too many of those for them to be weaponized. Once that is done, +1

File: hippiestation/code/game/objects/items/cheapcuisine.dm (hunk 1 above)
Comment (Carbonhell) [reply hippiestation/code/game/objects/items/cheapcuisine.dm]: Why would it add another component anyway, where else is it added

Comment (ghost) [reply hippiestation/code/game/objects/items/cheapcuisine.dm]: https://github.com/HippieStation/HippieStation/blob/master/code/modules/food_and_drinks/food/snacks_bread.dm#L277

**Comment by yoyobatty** [comment]
Ok, I'll take it off contraband too

File: hippiestation/code/game/objects/items/cheapcuisine.dm [hunk 2]
```diff
@@ -95,6 +95,10 @@
 	bitesize = 2
 	foodtype = GROSS | TOXIC
 
+/obj/item/reagent_containers/food/snacks/butterdog/carbon/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/slippery, 0)
```
Comment (Carbonhell) [inline hippiestation/code/game/objects/items/cheapcuisine.dm]: Proper way, I think, should be to GetComponent the component, then either qdel it or RemoveComponent it, not too sure yet

Comment (Carbonhell) [reply hippiestation/code/game/objects/items/cheapcuisine.dm]: yeah both do work, i guess qdel would make more sense though, qdel it

Comment (Carbonhell) [reply hippiestation/code/game/objects/items/cheapcuisine.dm]: D o I t

**Comment by Eagle-Eyess** [comment]
I kinda like the slip effect... but all in all it needs to be nerfed. +1

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
@yoyobatty did you test it

**Comment by yoyobatty** [comment]
Yes, it's ready!! @Carbonhell

**Pull request merged**